code/modules/power/lighting.dm

```diff
@@ -306,28 +304,6 @@
 					explode()
 			else
 				user << "<span class='warning'>This type of light requires a [fitting]!</span>"
-				return
-
-		// attempt to break the light
-		//If xenos decide they want to smash a light bulb with a toolbox, who am I to stop them? /N
-
-	else if(status != LIGHT_BROKEN && status != LIGHT_EMPTY)
-		user.changeNext_move(CLICK_CD_MELEE)
-		user.do_attack_animation(src)
-		if(W.damtype == STAMINA)
-			return
-		if(prob(1+W.force * 5))
```

Comment:
no more randomness when hitting a light, it has a specific health amount now (but the possible shock is still random)